CVChatbot/CVChatbot.Bot/ChatbotActions/Commands/Stats.cs

```diff
@@ -17,9 +17,9 @@ public override void RunAction(ChatExchangeDotNet.Message userMessage, ChatExcha
 
             var statsTable = doc.DocumentNode
                 .Descendants("table")
-                .Where(x => x.Attributes["class"] != null)
-                .Where(x => x.Attributes["class"].Value == "task-stat-table")
-                .Single();
+                .Single(x => 
+                    x.Attributes["class"].Value == "task-stat-table" && 
+                    x.Attributes["class"] != null);
```

Comment:
So, just asking, is a double "where" less efficient? My thought is it would act the same: short circuiting when it fails the first.